lib/pidfile/lib/pidfile.c

```diff
@@ -168,8 +168,8 @@ pidfile_deserialize(const char *home, struct pidfile *content)
     if (cJSON_IsNull(socket_path)) {
         memset(content->rest.socket_path, 0, sizeof(content->rest.socket_path));
     } else {
-        n = strlcpy(
-            content->rest.socket_path, cJSON_GetStringValue(socket_path), sizeof(content->rest.socket_path));
+        n = strlcpy(content->rest.socket_path, cJSON_GetStringValue(socket_path),
+                sizeof(content->rest.socket_path));
```

Comment:
Is there anyway to make the continuation indent use 4 in cases like this?

Reply:
Not that I'm aware of.

Reply:
Yeah I didn't see anything either. Oh well. Just something to live with.

Reply:
I changed it to match normal indent width, which is what most people would expect.